typescript/thing-description/package.json

```diff
@@ -0,0 +1,30 @@
+{
+  "name": "wot-thing-description-types",
+  "version": "1.1.0-26-July-2021",
```

Comment:
do we expect that this kind of versioning breaks somehow NPM since it does not follow their rules, see https://docs.npmjs.com/about-semantic-versioning

Having said that, adding SNAPSHOT doesn't seem to be problematic so it should not cause issues here.. I hope.

Reply:
Scripting Call 2021-08-09
* keep proposed  version scheme
* check-out whether it causes issues